src/models/db/account.js

```diff
@@ -44,10 +25,8 @@ class Account extends Model {
     if (data.balance) {
       data.balance = Number(data.balance)
     }
-    if (data.password) {
-      data.password_hash = hashPassword(data.password)
-      delete data.password
-    }
+
+    delete data.password_hash
```

Comment:
Oh, I see we delete the password in `setAccount` now. Might be a bit clearer to delete this in `convertToPersistent`.